libs/client/use-cases/part/list/shared/ui/src/lib/part-card/part-card-properties/part-card-properties.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="properties">
+  <div *ngFor="let property of partCardProperties" class="property">
+    <p class="tui-island__paragraph">
+      <span class="bold">Label: </span>{{ property.label }}
+    </p>
+    <p class="tui-island__paragraph">
+      <span class="bold">Value: </span>{{ property.value
+      }}<span class="bold">{{ property.unit }}</span>
```

Comment:
![prop](https://user-images.githubusercontent.com/23154054/215507344-4b409ec8-debd-4f9a-abad-7a0a50dc6e7f.PNG)

unless bolding the unit is not needed, it seems to me that a space between the value and the unit will improve readability